static/esp_variable.css

```diff
@@ -0,0 +1,600 @@
+:root {
+	--bg1: #eee;
+	--bg2: #cacaca;
+	--bg3: #07d;
+	--c1: #fff;
+	--bdc1: silver;
+	--bg4: #e8e8e8;
+	--bg5: #dbff0075;
+	--c2: #444;
+	--bg6: #fff;
+	--bdc2: #444 #444 #fff;
+	--c3: #000;
+	--bg7: #bdbdbd;
+	--bg8: #029b34;
+	--bg9: inherit;
+	--c4: inherit;
+	--c5: #07d;
+	--bg10: #07d;
+	--bg11: #f8f8f8;
+
+
+}
+
+[data-theme="dark"] {
+	--bg1: #444;
+	--bg2: #2e2e2e;
+	--bg3: #00000085;
+	--c1: #dfdfdf;
+	--bdc1: #545454;
+	--bg4: #2a2c2e;
+	--bg5: #553044;
+	--c2: #fff;
+	--bg6: #222;
+	--bdc2: transparent;
+	--c3: #fff;
+	--bg7: #00000030;
+	--bg8: #30553d;
+	--bg9: #222;
+	--c4: #c3c3c3;
+	--c5: #c3c3c3;
+	--bg10: #44607a;
+	--bg11: #44607a;
+
+}
+
+.checkmark:after,
+.dotmark:after {
+	content: '';
+	display: none;
+	position: absolute
+}
+
+.button,
+.menu {
+	text-decoration: none
+}
+
+.div_l,
+.menu {
+	float: left
+}
+
+.closebtn,
+.div_r {
+	color: #fff;
```

Comment:
I would expect *all* colors to be using `var(--colorname)` ?

Reply:
thanks. just added it

Reply:
I still see a lot of colors not using a `var(--colorname)`, even a few in the 'variable' part? Still confusing to me (and I have used `var()` in css before, css is just not in my daily workflow...).
AFAIK, when using `var()` there is no need to split off setting the colors in a separate 'section', they can be with the other attributes for that element, doing that will probably save some more space.

Reply:
> I still see a lot of colors not using a var(--colorname), even a few in the 'variable' part?

These are colors that don’t change, so they need no variable.

> AFAIK, when using var() there is no need to split off setting the colors in a separate 'section', they can be with the other attributes for that element, doing that will probably save some more space.

You are right. Haven’t thought about it because it was much easier to change the css based on the mediaquery section. I was not so easy to change everything without losing the oversight. Thanks for the hint!

Reply:
> was not so easy to change everything without losing the oversight.

It also makes it much easier to add extra styles later, like the `multi2row` class I intend to add for settings-tables having 2 rows per object, like in P123. When that css is merged (possibly via the other PR), I'll also apply that for 'tightly filled' plugins like P104 and P016 😉